lms/views/admin/application_instance.py

```diff
@@ -12,41 +12,6 @@
 from lms.views.admin import error_render_to_response, flash_validation
 
 
-class EmptyStringNoneMixin:
```

Comment:
Having two view makes this hack not needed anymore. It might be handy at some point. We can bring it back later.